### ISSUE TITLE
Local Web Server Crashes West Wind HTML unless run as administrator

### ISSUE THREAD

**Issue by littleearth** [RickStrahl/HelpBuilderReleases#64]
Version 5.1.0
Steps -> Build Output -> Local Web Preview

Application will crash. 

Appears to be a permission issue launching web server to work around run West Wind HTML as administrator.

Windows 10 (Creators Edition)

**Comment by RickStrahl** [comment]
Yup it's a permission problem - the Http port needs to access permissions in order to access it even locally. However, the setup program should be doing this automatically as part of the install process.

To fix this you can try one of two things:

* Start a Command Prompt with Run As Administration
* cd <HtmlHelpBuilderInstall> folder

Then run:

 ```
wwhelp.exe PERMISSIONS
```

Alternately you can do:

```
netsh http add urlacl url="http://*:64101/" user=Interactive
```

to just set the port.

wwhelp PERMISSIONS runs as part of setup, so I'm not sure why that isn't working. You might want to check the port used to match what comes up when it fails (or in your `wwhelp.config` file).

**Comment by RickStrahl** [comment]
Checking a little closer into the installer, I see that the `PERMISSIONS` call is not actually happening. That would explain the problem :-)

Thanks for reporting. I've added it in and it will be fixed in the next update (6.11).

**Comment by littleearth** [comment]
Hi,
This fixed the issue. Yeah the setup did not seem to do this on 2 PC's I have tested this on

netsh http add urlacl url="http://*:64101/" user=Interactive

Do you use Innosetup for the installation script? 

Example I have used.

Filename: "Netsh"; Parameters: "advfirewall firewall add rule name=""{#ConstAppName}"" dir=in action=allow program=""{app}\{#ConstAppExeName}"" enable=yes profile=any"; Flags: runhidden; StatusMsg: "Updating Windows Firewall..."; Tasks: modifyfirewall

**Comment by littleearth** [comment]
No worries. Thanks for prompt reply.

**Comment by RickStrahl** [comment]
Yeah I switched to Inno recently due to AV flagging by my old installer. The PERMISSIONS call was something I forgot to port. There are a few other things that get set there and that might actually cause some other issues.

**Comment by RickStrahl** [comment]
This was fixed in 6.11 a while back.
